management-gateway/src/main/java/org/frankframework/management/gateway/HazelcastInboundGateway.java

```diff
@@ -52,9 +55,13 @@ public class HazelcastInboundGateway extends MessagingGatewaySupport {
 	private final String requestTopicName = HazelcastConfig.REQUEST_TOPIC_NAME;
 	private ITopic<Message<?>> requestTopic;
 
+	@Value("${instance.name:}")
```

Comment:
Does the `:` do something special?

Reply:
Ensures the default value is empty and not null.